packages/contracts/scripts/write-version.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { version } = require('../package.json')
```

Comment:
nit: Can we use the standard import syntax here? i.e. `import { version } from '../package.json'`